app/cisc/lib.go

```diff
@@ -11,50 +11,148 @@ import (
 
 	"golang.org/x/crypto/ssh"
 
+	"strings"
+
 	"github.com/dedis/cothority/app/lib/config"
 	"github.com/dedis/cothority/log"
 	"github.com/dedis/cothority/network"
 	"github.com/dedis/cothority/services/identity"
 	"gopkg.in/codegangsta/cli.v1"
 )
 
+func init() {
+	network.RegisterPacketType(ciscConfig{})
+}
+
+type ciscConfig struct {
+	*identity.Identity
+	Follow []*identity.Identity
+}
+
 // loadConfig will try to load the configuration and fail if it can't load it.
-func loadConfig(c *cli.Context) *identity.Identity {
+func loadConfig(c *cli.Context) (*ciscConfig, bool) {
```

Comment:
In this case it might be useful if the return statements were named like `(cfg *ciscConfig, empty bool)` and just use return. One needs to search for callers to find out what this `bool` means.